models/models.js

```diff
@@ -11,3 +11,30 @@ exports.fetchTopics = () => {
       return result.rows
     })
 };
+
+exports.fetchArticles = () => {
+  return db
+    .query(
+    `
+    SELECT 
+      articles.author, 
+      articles.title, 
+      articles.article_id, 
+      articles.topic, 
+      articles.created_at, 
+      articles.votes, 
+      articles.article_img_url,
+      CAST(COUNT(articles.article_id) AS INT) AS comment_count
+    FROM articles
+    JOIN comments
```

Comment:
You might need a LEFT JOIN here - depends if you're getting back what you expect.